doc/topics/cloud/azurearm.rst

```diff
@@ -95,7 +97,7 @@ Set up an initial profile at ``/etc/salt/cloud.profiles``:
       size: Standard_D1_v2
       location: eastus
       ssh_username: azureuser
-      ssh_password: verybadpass
```

Comment:
`ssh_password` should not be removed from the example, since it is still supported. How about another example block for `ssh_publickeyfile`?